settings.py

```diff
@@ -584,7 +584,9 @@
 # Keep messages in the events queue only for 2 hours
 CELERY_EVENT_QUEUE_TTL = 2 * 60 * 60
 
-CELERY_TASK_SERIALIZER = 'json'  # Default value in celery 4.x
+# Default serializer should be changed back to 'json' after
+# https://github.com/celery/celery/issues/6759 is fixed
+CELERY_TASK_SERIALIZER = 'pickle'
```

Comment:
An inline comment will make this more visible when searching with grep
```suggestion
CELERY_TASK_SERIALIZER = 'pickle'  # default is 'json'. See corehq.apps.celery.shared_task
```